urls.js

```diff
@@ -0,0 +1,41 @@
+const fs = require('fs');
+const process = require('process');
+const axios = require('axios');
+
+function getUrls(path) {
+  fs.readFile(path, 'utf8', (err, data) => {
+    if (err) {
+      console.error(`Error reading ${path}: ${err}`);
+      process.exit(1);
+    } else {
+      const urls = data.split('\n').filter((url) => {
+        return url !== '';
+      });
+      getUrlData(urls);
+    }
+  });
+}
+
+function getUrlData(urls) {
```

Comment:
Should be async and await `Promise.all` of `urls.map` instead of `forEach`. Then in `getUrls`, `await getUrlData(urls)` so you wait for downloads to finish before exiting.